scripts/env.sh

```diff
@@ -0,0 +1,7 @@
+export WEB3_SYSTEM_TEST_ENV=true
+export WEB3_SYSTEM_TEST_BACKEND=""
+export WEB3_SYSTEM_TEST_PASSWORD="smart guide what forget tired jungle always expire rescue boring glue champion"
```

Comment:
I would change this to `WEB3_SYSTEM_TEST_MNEMONIC` as there's a notion of a `password` for encrypting the private key